### DISCUSSION TITLE
Microphone not working

### DISCUSSION THREAD

**Discussion by djvo** [HomeSpan/HomeSpan#718] [Q&A]
Hey, 
First of all thanks for this great library. I like to use a microphone to measure a volume over a certain time and use it like a sensor to open a window. But there is no example or decription for it. Is the mircophone working? It always says in homekit that it's not supported.
Thanks in advance for more information!

**Comment by HomeSpan** [comment]
I just tested and it seems that the Home App no longer supports the microphone service, at least on a standalone basis.  It may be that it works but only if included as part of the audio stream service (which are not supported by HomeSpan).

However, I'm not sure you need to use the microphone service for what you wish to accomplish.  The only thing it allows you do to (if it worked) would be request mute/unmute the mic from within the Home App.  It does not provide any actual microphone functionality.
